src/sparseml/sparsification/modifier_epoch.py

```diff
@@ -17,7 +17,10 @@
 model
 """
 
+from typing import List
+
 from sparseml.optim.modifier import BaseModifier, BaseScheduled
+from sparseml.sparsification.types import SparsificationTypes
```

Comment:
nit: For consistency
```suggestion
from sparseml.sparsification import SparsificationTypes
```

Reply:
since this is in the same package, we run the risk of a circular dependency being introduce in the future if we inherit from the parent init that we export to